docs/topics/internationalization.md

```diff
@@ -43,7 +43,7 @@ REST framework includes these built-in translations both for standard exception
 
 Note that the translations only apply to the error strings themselves. The format of error messages, and the keys of field names will remain the same. An example `400 Bad Request` response body might look like this:
 
-    {"detail": {"username": ["Esse campo deve ser único."]}}
+    {"detail": {"username": ["Esse campo deve set único."]}}
```

Comment:
Please revert.  This is NOT English.

Reply:
thanks for the catch, missed that, decided to just ignore lines with  ú